eclass/bitcoincore.eclass

```diff
@@ -159,7 +162,7 @@ bitcoincore_common_depend_use() {
 	in_bcc_iuse "$1" || return
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $1? ( $2 )"
 }
-bitcoincore_common_depend_use upnp net-libs/miniupnpc
+bitcoincore_common_depend_use upnp '>net-libs/miniupnpc-1.9.20150915'
```

Comment:
(in other words, what needs to be done to restore security here is removal of all versions prior to 1.9.20150916, and keywording of one or more of the secure versions)